h2o-extensions/xgboost/src/main/java/hex/schemas/XGBoostV3.java

```diff
@@ -55,7 +55,6 @@
 
         "score_tree_interval",
         "min_split_improvement", "gamma",
-        "auc_type",    
```

Comment:
@maurever this one was duplicated, which created issues with PY code generation.
I kept the one at the end because that's where it is for most algos. Is it better to have here after `gamma`? (I don't really see why)

Reply:
Good catch @sebhrusen! That is my mistake. I did not remember why I added it twice. It is ok to remove this one. Thank you!